src/Global/atoms/UserDrawerMeta.tsx

```diff
@@ -75,23 +76,100 @@ const Container = styled.div`
   cursor: pointer;
 `;
 
+const TitleBox = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const IconBox = styled.div`
+  padding: 1px 5px 0 0;
+`;
+
+const CopyTextBox = styled.div<{ tooltipValue: boolean }>`
+  text-align: center;
+  padding: 5px;
+  font-size: 10px;
+  line-height: 1.2;
+  box-shadow: 2px 2px 4px (195deg 16% 72% / 72%);
+  border: 1px solid hsl(195deg 5% 60% / 72%);
+  background-color: #f8f9fa;
+  border-radius: 5px;
+  opacity: 0.76;
+  font-size: 10px;
+  font-weight: 500;
+  color: #575757;
+  ${({ tooltipValue }) => tooltipValue
+    ? `margin-left: -9%;`
+    : `margin-left: -27%;`
+  };
+  margin-top: -23px;
+  position: absolute;
+  ${({ tooltipValue }) => tooltipValue
+    ? `min-width: 45px;`
+    : `min-width: 82px;`
+  };
```

Comment:
I think this will not be needed. It should change the layout based on content size.